docs/data/onboarding.md

```diff
@@ -0,0 +1,85 @@
+# Onboarding new datasets
+
+Adding new datasets to the data lake usually involves three steps:
+
+1. Add the new data to the [`Raw` bucket](https://github.com/cds-snc/data-lake/blob/main/terragrunt/aws/buckets/raw.tf).
```

Comment:
should we mention it should be `unclassified data` (until we get an ATO for PB) ?

Reply:
Good idea - I've updated the doc to mention that.